src/background/sidebar-injector.js

```diff
@@ -39,19 +39,17 @@ function addJSONScriptTag(name, content) {
  * chrome.tabs.executeScript() into the main frame of a page.
  *
  * executeScript() returns an array of results, one per frame which the script
- * was injected into.
+ * was injected into. If the frame fails to execute the script it returns a value
+ * of `null`.
```

Comment:
```suggestion
 * was injected into. If a frame fails to execute the script the array entry will
 * have a value of `null`.
```

I reworded this to clarify that the `null` refers to entries in the `result` array, not `result` itself.